symphony-bdk-core/src/main/java/com/symphony/bdk/core/OboServicesFacade.java

```diff
@@ -0,0 +1,57 @@
+package com.symphony.bdk.core;
+
+import com.symphony.bdk.core.auth.AuthSession;
+import com.symphony.bdk.core.client.ApiClientFactory;
+import com.symphony.bdk.core.config.model.BdkConfig;
+import com.symphony.bdk.core.service.message.OboMessageService;
+import com.symphony.bdk.core.service.stream.OboStreamService;
+import com.symphony.bdk.core.service.user.OboUserService;
+
+import org.apiguardian.api.API;
+
+/**
+ * Entry point for OBO-enabled services.
+ */
+@API(status = API.Status.EXPERIMENTAL)
+public class OboServicesFacade {
```

Comment:
Now I'm wondering if `OboServicesFacade` is the right name 🤔

Reply:
Just OboServices?